packages/utils/src/anr.ts

```diff
@@ -0,0 +1,110 @@
+import type { StackFrame } from '@sentry/types';
+import type { Debugger } from 'inspector';
```

Comment:
I don't think we can do this because utils can't use node types 😢.

Would be nice if we had submodule exports and could just export it from `@sentry/node/utils`

Reply:
Won't `import type` be completely removed from the output files?

Reply:
It's not removed from the output, and it does cause issues for some people's tsconfig setups.

Reply:
Ok, I'll extract the types!